### ISSUE TITLE
Sort CSS properties alphabetically

### ISSUE THREAD

**Issue by marcobiedermann** [picturepan2/spectre#12]
Sorting CSS properties by alphabet will reduce the file size when gzipped:

http://peteschuster.com/2014/12/reduce-file-size-css-sorting/

**Comment by picturepan2** [comment]
thanks. doing now.

**Comment by picturepan2** [comment]
I also added CSScomb to the Gulp process. The minified css file is reduced by 1157 byte.

**Comment by marcobiedermann** [comment]
I've seen that.
But I think it is also a good idea to sort the CSS properties in all LESS files.
This way, spectre does not depend on a third party tool like CSSComb.
What do you think?

**Comment by picturepan2** [comment]
Yes. I was thinking about this - my current coding order is like position > size > box model > text etc. It is hard to follow perfectly. Since CSSComb doesn't support LESS, do you recomend any automation tool for this?

**Comment by marcobiedermann** [comment]
Not automated but [stylelint](https://github.com/stylelint/stylelint), now supports LESS files.

**Comment by picturepan2** [comment]
Cool. Thanks. Hope I will get to used to this new habit. Lol.

**Comment by picturepan2** [comment]
Done. I updated all LESS files by using CSSComb. I still keep CSSComb in Gulp process because Autoprefixer adds new properties.
